README.md

```diff
@@ -130,21 +130,18 @@ public class MainActivity extends ReactActivity {
 pod install --project-directory=ios/
 ```
 
-2. Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` into that directory.
+2. Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.framework` into that directory.
 
-3. Add `Indy.framework` as dependency into your project. Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk. Beware that `Indy.framework` needs to be compiled with the same version of Swift as your project.
+3. [Optional, Recommended] Most projects have `ios/Pods` ignored in their `.gitignore`. This is good practice, however this means the framework would need to be added after every clone. To prevent this you can "unignore" the Frameworks directory:
```

Comment:
I intentionally didn't add this step 😄  I'm ok to leave add it as Optional, but I wouldn't mark it as "Recommended". I think you shouldn't add a binary file into your project repo until it's absolutely necessary. Although you can eventually remove it by some git juggling, I don't think it's easy to do so.

Reply:
Fair point. I'll remove the "Recommended". Seeing how bad indy is distributed, I'd personally add it to the repo. The pod is not working.

Reply:
Updated